src/index.html

```diff
@@ -4,10 +4,146 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./css/all.css">
+    <link rel="stylesheet" href="./styles/reset.css">
     <link rel="stylesheet" href="./styles/main.css">
+    <link rel="icon" href="images/favicon.png">
   </head>
   <body>
-    <h1>Miami</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+      <header class="header">
+        <div class="wrapper">
+          <div class="header__top">
+            <div class="header__phone">+1 234 555-55-55</div>
+            <a href="tel:+380731563436"><img class="header__phone-icon" src="./images/phone.svg" alt=""></a>
+            <div class="logo header__logo"></div>
+            <input type="checkbox" name="toggler" id="menu" class="togglerMenu">
+            <label for="menu" class="header__hamburg-icon"></label>
+            <ul class="mobile-nav">
+              <li class="mobile-nav__item"><a href="#" class="mobile-nav__link">Home</a></li>
```

Comment:
```suggestion
              <li class="mobile-nav__item">
                <a href="#" class="mobile-nav__link">Home</a>
              </li>
```
улучшаем читаемость (исправь везде где можно)